docs/modules/devicetwin.md

```diff
@@ -77,32 +77,39 @@ The following are the action callbacks which can be performed by the membership
    - dealMembershipUpdated
    - dealMembershipDetail
 
-**dealMembershipGet**:   dealMembershipGet() deal gets the information  about the devices associated with the
-                                                                    particular edge node, from the cache. The eventbus first receives a message on its subscribed topic (membership-get topic).
-                                                                    This message arrives at the  devicetwin controller, which further sends the message to membership module.  The membership module
-                                                                     gets the devices associated with the edge node from the cache (context) and sends the information to the communication module, 
-                                                                     it also handles errors that may arise while performing the  aforementioned process and sends the error to the communication module
-                                                                      instead of device details.The communication module sends the information to the  eventbus component which further publishes the result on the 
-                                                                     specified MQTT topic (get membership result topic). 
+**dealMembershipGet**:    dealMembershipGet() deal gets the information  about the devices associated with the
```

Comment:
dealMembershipGet() ~~deal~~ gets the information  about

Reply:
Done, thanks